### DISCUSSION TITLE
Menu item to choose from all available screen resolution and refresh rates

### DISCUSSION THREAD

**Discussion by asherlporetz** [rxhanson/Rectangle#1015] [General]
It seems related enough to Rectangle's functionality to want to set the resolution, refresh rate, and even screen orientation. Can this be added? Mac tends to hide a lot of modes that a monitor is actually capable of, especially higher refresh rate at native resolution.

**Comment by rxhanson** [comment]
Thanks for the feedback. Rectangle is a window management tool. I would expect this type of functionality to be included in a display management tool instead.
